components/search/src/selector.rs

```diff
@@ -247,6 +247,23 @@ mod tests {
                         "base": "https://example.com/1",
                         "method": "GET",
                         "searchTermParamName": "q"
+                      },
+                      "suggestions": {
+                        "base": "https://example.com/suggestions",
+                        "method": "POST",
+                        "params": [{
+                          "name": "type",
+                          "value": "space",
```

Comment:
Wondering if we could we name these to `suggestion-name` and `suggestion-value`?